src/Microsoft.Extensions.ApplicationModelDetection/AppModelDetectionResult.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.ApplicationModelDetection
```

Comment:
If this can detect only ASP.NET Core apps, is `Microsoft.Extensions.ApplicationModelDetection` a correct name for it? Should it be `Microsoft.AspNetCore.ApplicationModelDetection`?